tests/test_backends.py

```diff
@@ -83,6 +84,47 @@ def test_execute_circuits():
         assert sum(result.frequencies().values()) == 100
 
 
+def test_multiple_measurements():
+    backend = QibolabBackend("dummy")
+
+    circuit = Circuit(4)
+    circuit.add(gates.H(i) for i in range(2))
+    circuit.add(gates.CZ(0, 1))
+    res0 = circuit.add(gates.M(0))
+    res1 = circuit.add(gates.M(3))
+    res2 = circuit.add(gates.M(1))
+    result = backend.execute_circuit(circuit, nshots=50)
+
+    samples = [getattr(res, "samples")()[:, 0] for res in [res0, res1, res2]]
```

Comment:
```suggestion
    samples = [res.samples()[:, 0] for res in [res0, res1, res2]]
```
If you know the name, just use it ;)